packaging/build.mjs

```diff
@@ -27,6 +27,23 @@ await packContractsAndChallenges()
 
 const { version, revisionIdent } = require("../package.json")
 
+const jsonAsCompressedTextPlugin = {
+    name: "jsonAsCompressedTextPlugin",
+
+    setup(build) {
+        let fs = require("fs")
+
+        build.onLoad({ filter: /\.json$/ }, async (args) => {
+            const text = await fs.promises.readFile(args.path)
+
+            return {
+                contents: JSON.stringify(JSON.parse(text)),
+                loader: "text",
+            }
+        })
+    },
+}
```

Comment:
I think this can just use the [`loader` option](https://esbuild.github.io/api/#loader) instead of making this a plugin.

Reply:
Nope, that was my first attempt and it will only treat it as text, and not actually remove indentation like I do now. The JSON loader will turn it into an object, so this is the only way to do it and also "compress" the actual resulting string :)

Reply:
`let fs = require("fs")` I want to change this to const though. Using an import doesn't work do to the way esbuild works with plugins.